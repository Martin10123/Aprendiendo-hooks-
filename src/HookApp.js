import React from "react";

export const HookApp = () => {
  return (
    <div className="App">
      <h1>hola mundo!!</h1>
    </div>
  );
};
