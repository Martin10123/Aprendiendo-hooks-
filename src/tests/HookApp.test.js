import React from "react";
import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe("Pruebas en el componente <HookApp/>", () => {
  test("Debe funcionar correctamente", () => {
    const wrapper = shallow(<HookApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
