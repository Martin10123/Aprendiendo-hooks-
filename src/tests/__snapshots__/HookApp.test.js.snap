// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <HookApp/> Debe funcionar correctamente 1`] = `
<div
  className="App"
>
  <h1>
    hola mundo!!
  </h1>
</div>
`;
